Detectors/EMCAL/simulation/include/EMCALSimulation/Digitizer.h

```diff
@@ -73,7 +80,9 @@ class Digitizer : public TObject
 
   void setGeometry(const o2::emcal::Geometry* gm) { mGeometry = gm; }
 
-  Digit hitToDigit(const Hit& hit, const Int_t label);
+  void hitToDigits(const Hit& hit, std::vector<Digit>& digits);
+
+  static double rawResponseFunction(double* x, double* par);
```

Comment:
would it be possible to avoid raw pointers here? What is x and what is par? Could you use std::array<> or some other strong type?

Reply:
If then gsl::span<double>. However the function should also work with the ROOT TF1 interface (i.e. for fits), and therefore I am not sure if gsl::span works.

Reply:
Ok. But I guess one can retrieve the raw pointer inside (and still declare the interface type-safe)